boards/arm/mimxrt1050_evk/Kconfig.defconfig

```diff
@@ -94,4 +94,18 @@ endchoice
 
 endif # LVGL
 
+if CHARACTER_FRAMEBUFFER
+
+config  HEAP_MEM_POOL_SIZE
+	default 300000
+
+config CHARACTER_FRAMEBUFFER_DISPLAY_DEV_NAME
```

Comment:
We should be get name from DT not from Kconfig like this.

Reply:
@galak I think not in that case, the problem here is the lacking of display interface registration/discovery. I am on it.

Reply:
@galak Agree here with @jfischer-phytec-iot currency this the only way of telling  which display should be used by CFB.

Reply:
so no change need right? @galak